tools/cldr-apps/src/main/webapp/tc-all-errors.jsp

```diff
@@ -0,0 +1,109 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
```

Comment:
Could we avoid adding any new jsp files? It's a legacy technology and we should be getting rid of the ones we have rather than adding new ones

Reply:
OK. They are easier to create for simpler pages like this- but, I will rewrite it.

Reply:
decided to not rewrite(for this PR), could be done as a followon